commands/backfill.js

```diff
@@ -113,7 +113,7 @@ module.exports = function container (get, set, clear) {
                 console.error('\nerror: getTrades() returned duplicate results')
                 console.error(opts)
                 console.error(last_batch_opts)
-                process.exit(1)
+                process.exit(0)
```

Comment:
I'm wondering if this is a good idea:
bittrex doesn't support historical/specific fetching of data, so we need to fetch every minute, but this can still mean we get duplicates (or miss data), exiting with `0` means that we still continue with running paper/live trading.